gno.land/pkg/sdk/vm/msgs.go

```diff
@@ -57,7 +57,7 @@ func (msg MsgAddPackage) ValidateBasic() error {
 		return ErrInvalidPkgPath("missing package path")
 	}
 	if !msg.Deposit.IsValid() {
-		return std.ErrTxDecode("invalid deposit")
+		return std.ErrInvalidCoins("invalid deposit")
```

Comment:
most other places using ErrInvalidCoins do it with `std.ErrInvalidCoins(amt.String())` -- so please do the same

Reply:
ff3ad2350f15e897488baa4cbf2ea261695a2899